app/src/main/java/org/oppia/android/app/player/state/answerhandling/InteractionAnswerHandler.kt

```diff
@@ -26,6 +28,11 @@ interface InteractionAnswerHandler {
   fun getPendingAnswer(): UserAnswer? {
     return null
   }
+
+  /** Returns the current answer. */
```

Comment:
Is "curent selected answer" more descriptive?

Reply:
`selected` term align with only selection type interactions. That's the reason i kept as above.

Reply:
Ah I see. Maybe use  "pending answer" to show that it is the input that has not been submitted.

Reply:
Okay.